features/cellular/framework/AT/AT_CellularSMS.cpp

```diff
@@ -587,7 +611,7 @@ nsapi_error_t AT_CellularSMS::delete_all_messages()
 
 // read msg in text mode
 nsapi_size_or_error_t AT_CellularSMS::read_sms_from_index(int msg_index, char *buf, uint16_t len, char *phone_num,
-                                                          char *time_stamp)
```

Comment:
Suggest putting the spaces back as they were to avoid nit-pick comments.

Reply:
I run astyle.exe, once suggested by ARM guy on my previous PR, for formatting. This styling removed these spaces there. I will do restore it manually.